react-components/my-app/src/ImageContainer.tsx

```diff
@@ -0,0 +1,3 @@
+export function ImageContainer() {
+  return <img src="https://images.app.goo.gl/u45i3DrVJDJhmjb69" />;
```

Comment:
This link is not an actual image. In Google Images with Google Chrome, you can right-click the image and find "Copy Image Address". That's the URL you want to use.